docs/HyperIndex/Advanced/config-schema-reference.md

```diff
@@ -206,7 +208,7 @@ field_selection:
   transaction_fields:
     - hash
   block_fields:
-    - timestamp
+    - miner
```

Comment:
This was just wrong, Timestamp is always included on the block and if you add this here the codegen will not allow it (and fail with error)